packages/messaging/lib/index.d.ts

```diff
@@ -611,12 +611,12 @@ export namespace FirebaseMessagingTypes {
     getToken(authorizedEntity?: string, scope?: string = 'FCM'): Promise<string>;
 
     /**
-     * Returns wether the root view is headless or not
+     * Returns wether the root view launched was headless or not
```

Comment:
I think
```
Returns `true` if the root view was launched as a headless JS task, `false` otherwise.
```
is a bit more clear

Reply:
other than that I really like it. Simple, and easy to use. 🎉 

 I just hope we don't run into any trouble with using a constant, e.g. when the JS context gets destroyed, application is in background, or other specific cases.